### ISSUE TITLE
Chest unlootable for Unfinished Gordok Business quest

### ISSUE THREAD

**Issue by vidiskokan** [dalaranwow/dalaran-wow#5899]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Chest that is supposed to be lootable after killing Prince Tortheldrin for [Unfinished Gordok Business](https://classic.wowhead.com/quest=7703/unfinished-gordok-business) is not lootable.

**Expected behaviour**: Chest should be lootable after killing Prince Tortheldrin

You guys addressed this [issue](https://github.com/dalaranwow/dalaran-wow/issues/2503) already in 2016, you said its fixed, but it is again an issue.

**Steps to reproduce the problem**:

1. Finish DM tribute run.
2. Talk to Captain Kromcrush to get the quest.
3. Reset instances
4. Go to DM West, kill [Prince Tortheldrin](https://classic.wowhead.com/npc=11486/prince-tortheldrin) 
5. Below stairs is the chest with loot that should be lootable when you kill prince, but it is not

**Include proofs for this behaviour**

[Image to prove](http://korelacija.com/image.jpg) (note that cursor is over chest but its not turned into action cursor - can't loot)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
